.github/workflows/ci.yml

```diff
@@ -37,6 +37,10 @@ jobs:
            mcVersion: '1.19.3'
          - javaVersion: 17
            mcVersion: '1.19.4'
+         - javaVersion: 17
+           mcVersion: '1.20'
+         - javaVersion: 17
```

Comment:
Since there is no protocol change in 1.20.1 and 1.20, just testing 1.20.1 should save time than doing the same tests twice, to match how previous versions are tested. But I guess that can be handled later.

Reply:
Yup, it was helpful to have that originally just to be sure everything worked on both in case Mojang did something weird that wasn't in the changelog, but definitely makes sense to only test one once this is merged